composer.json

```diff
@@ -19,7 +19,7 @@
     ],
     "require": {
         "akeneo/pim-community-dev": "2.*",
-        "akeneo-labs/custom-entity-bundle": "v2.0.2-beta",
+        "akeneo-labs/custom-entity-bundle": "2.0.*",
```

Comment:
2.*?
We are already in 2.1 for this extension and I will release the 2.2 soon (with export fixes)